BackendNew/resources/openapi.yaml

```diff
@@ -1212,6 +1212,37 @@ paths:
 
 components:
   schemas:
+    BasicStudent:
+      type: object
+      properties:
+        id:
+          type: string
+          description: "The index number of the student"
+        batch:
+          type: string
+          description: "The intern batch of the student"
+        fullName:
```

Comment:
Did we agreed upon to add both full name and name with initials by admin at initial phase?

Reply:
Will make it as Name with initials